### DISCUSSION TITLE
Join NUMFOUCS

### DISCUSSION THREAD

**Discussion by tkoyama010** [pyvista/pyvista#2935] [Ideas]
I put this up as an idea.  [NUMFOCUS ](https://numfocus.org/) is a non-profit organization promoting accessible and reproducible scientific and technical computing.
Drawing tools such as [Matplotlib](https://numfocus.org/project/matplotlib) also receive support.
I want to get your opinion on joining this.

https://numfocus.org/projects-overview
![image](https://user-images.githubusercontent.com/7513610/177077437-b47a0605-32c2-48a2-98cc-a5080d02dcd7.png)

**Comment by akaszynski** [comment]
At PyCon 2022 I spoke with representatives from NumFOCUS and it seemed like an excellent idea. Fully agree that we should apply.

**Reply by tkoyama010** [reply]
Great! I will wait a day for other opinions and if there are no objections I will apply to join.

**Reply by banesullivan** [reply]
+1 for applying - I looked into this about a year ago but was ultimately too busy to follow through.

My understanding is that it can be a great way to get small development grants and support contributors who have been volunteering time or fund the development of larger features that the community wants

**Reply by banesullivan** [reply]
@tkoyama010, let me know on Slack when you start doing this - I would like to be involved

**Reply by tkoyama010** [reply]
Thanks @banesullivan I will.
